Pidaq/Lib/uploadToDrive/uploadToDrive.py

```diff
@@ -0,0 +1,118 @@
+        parentDictionary = json.load(json_file)
+except FileNotFoundError:
+    logging.error('Parents dictionary .json file not found')
+    raise SystemExit
+# Change this for the ID of the default folder 
+DefaultID = '1VDOOjuOeyNRJgdKmmRTJC1HVjq91F4Q8'
+
+def uploadCsv(filePath, parentName):
+    """ Uploads a csv file from a local directory to a folder on Google Drive.
+
+    Parameters:
+    "filePath" is the path of the csv file itself and
+                should be passed without quotations.
+    "parentName" is the name of the testing folder in the drive
+            and only a specific set of values are allowed.
+    The parent folder is supposedly going to be fixed for a
+    long period of time.
+    """
+    # Searches for the parent name in the dictionary
+    for key in parentDictionary.keys():
+        if key.lower() == parentName.lower():
+            parentKey = key
+    try:
+        parentFolderID = parentDictionary[parentKey]
+    except NameError:
+        # Uploads to a default folder if the folder name does
+        # Not exist in the dictionary
+        parentFolderID = DefaultID
+        logging.error('Folder name not found \nuploading to "Default"')
+    
+    # Today's date in YYYY-MM-DD format is stored in the variable "date"
+    date = datetime.today().strftime('%Y-%m-%d')
+    
+    # If the folder does not exist, should be the case
+    # When running for the first time on a given day
+    if checkExistingSubFolder(date, parentFolderID) == 0:
+        folder_metadata = {
+            'name' : date,
+            'mimeType': 'application/vnd.google-apps.folder',
+            'parents' : [parentFolderID]
+            }
+        # Creates a folder with the given date
+        folder = DRIVE.files().create(body = folder_metadata,
+                                      fields = 'id').execute()
+        # Returns the date folder's ID
+        folderId = getFolderId(date, parentFolderID)
+    # Executes if the folder already exists  
+    elif checkExistingSubFolder(date, parentFolderID) == 1:
+        logging.warning("found folder for today's date")
+        folderId = getFolderId(date, parentFolderID)
+    # Executes if more than one folder exists
+    else:
+        logging.warning('unexpected count of folders named with a given date')
+        return
+    
+    # Extracts the .csv file name from the path
+    filename=os.path.basename(filePath)
+    metadata = {'name': filename, 'parents' : [folderId] }
+    # Uploads the .csv file
+    response = DRIVE.files().create(body=metadata,
+                               media_body=filePath).execute()
+    
+    if response:
+        logging.warning('Uploaded "%s" (%s)' % (metadata['name'], response['mimeType']))
+    else:
+        logging.warning('error in uploading the csv file')
+        
+def checkExistingSubFolder(folderName, parentFolder):
+    """ Queries the drive for the folder with the given name and parent folder
+    """

```

Comment:
One line docstrings have the quotations on the same line 

""" XXXXXXXXXXXXXXX """